transforms/zome_function.js

```diff
@@ -0,0 +1,121 @@
+const assemblyscript = require('assemblyscript');
+const {
+  CommonFlags,
+  Node,
+  NodeKind,
+  SourceKind,
+  TypeKind,
+  parseFile
+} = assemblyscript;
+
+
```

Comment:
The same would go for here, but that's not the work of this PR, and could be done in a separate one